src/explanations/usdt.yaml

```diff
@@ -0,0 +1,65 @@
+
+
+    However, Tether's Taiwanese banking partners grew concerned about the lack of know-your-customer (KYC) checks and other safeguards as the stablecoin circulated on various blockchains. They ended their relationship with Tether in 2017, which jeopardized Tether's ability to facilitate the conversion of fiat currency to cryptocurrency and vice versa. Tether was able to secure new banking partners in Puerto Rico and the Bahamas, and during the subsequent cryptocurrency bull market, Tether's stablecoins (others being EURT, CNHT, MXNT, XAUT) became widely used, with its flagship USDT token ranking among the top tokens by traded volume and market capitalization.
+
+
+    Tether was originally built on the Bitcoin network, but has since expanded to other leading blockchains. It is registered in the British Virgin Islands and lists JL van der Velde as its CEO. Tether and Bitfinex are both owned by iFinex.
+
+
+    ### Mint and Burn
+
+
+    Tether Limited issues USDT by "minting" new tokens and adding them to circulation. This is done through a process called "issuance," which involves Tether Limited sending the stablecoin to exchanges or other approved parties in exchange for an equal value of the stable asset (e.g., US dollars). Tether Limited can also "burn" USDT by removing them from circulation. This is typically done when the company receives the stablecoin back from exchanges or other approved parties in exchange for the stable asset.
+
+
+    ### Transparency and Audits
+
+
+    Tether Limited has faced criticism over its lack of transparency, as the company has not undergone a full audit to prove that it has the necessary reserves to back all of the USDT in circulation. In 2017, Tether Limited hired accounting firm Friedman LLP to conduct an audit, but the audit was never completed and the relationship between the two companies ended in January 2018. 
+
+
+    Tether has been publishing quarterly audit [reports](https://tether.to/en/transparency/#reports) since 2021 with the help of firms such as Moore, Mha Cayman and BDO. However, most of the companies feel that those are just attestation which don't give the full picture.
+
+
+    ### How They are Backed
+
+
+    USDT is meant to be backed 1:1 by the stable asset, meaning that for every USDT in circulation, Tether Limited is supposed to hold an equivalent value of the stable asset in reserve. According to Tether Limited, the stable asset backing USDT is typically held in the form of US Treasury bills, which are low-risk, short-term debt securities issued by the US government. However, as mentioned above, the company has not undergone a full audit to prove that it has the necessary reserves to back all of the USDT in circulation.
+
+
+    ### Issues and Hacks
```

Comment:
The below three sections seems to be the same i.e. 
1) Transparency and Audits
2) Issues and Hacks
3) Good and Bad News 

and can be consolidated into one